docker/dev.Dockerfile

```
@@ -27,21 +32,21 @@ RUN ARCH=$(dpkg --print-architecture) && \
-RUN pip install pipenv==2024.2.0
+RUN pip install pipenv
 
-COPY Pipfile Pipfile.lock ./
-RUN pipenv install --system --categories "packages dev-packages"
+RUN mkdir -p $APP_HOME/.venv
+COPY Pipfile Pipfile.lock $APP_HOME/
+RUN --mount=type=cache,target=/root/.cache/pip pipenv sync --categories "packages dev-packages docs"
 
-COPY . /app
+COPY plugs/ $APP_HOME/plugs/
+COPY install_plugins.py plug_config.py $APP_HOME
```

Hadolint

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)